env/snake/script.sh

```diff
@@ -1,31 +1,9 @@
-#!/bin/bash
-
-echo -------------- Update source list --------------
-sudo apt-get update -y
-

```

Comment:
I am not sure if we will want to delete this file as Mehdi is using it to provision the machine on windows. 

Annoyingly We may need two different scripts, if you can copy the previous script to a new file once it is merged I can run it and see if we do need it.